Systems/electrical.xml

```diff
@@ -12,5 +12,19 @@
             <property>/systems/electrical/battery-charge-percent-100</property>
         </output>
     </filter>
-
+    
+    <!-- ============================================================== -->
+    <!-- Save state of aircraft between sessions                        -->
+    <!-- ============================================================== -->
+    
+    <logic>
+        <name>Save state of aircraft between sessions => save battery charge</name>
+        <input>
+            <property>/sim/model/c172p/save-state</property>
+        </input>
+        <output>
```

Comment:
Okay, please let me know, but I did test it and only the very first property was affected...